docs/List.md

```diff
@@ -672,15 +672,15 @@ The title can be either a string or an element of your own.
 
 ## `sx`: CSS API
 
-The `List` component accepts the usual `className` prop but you can override many class names injected to the inner components by React-admin thanks to the `sx` property (as most Material UI components, see their [documentation about it](https://mui.com/customization/how-to-customize/#overriding-nested-component-styles)). This property accepts the following subclasses:
-
-| Rule name                        | Description                                                                              |
-| -------------------------------- | ---------------------------------------------------------------------------------------- |
-| `&.RaList-root`                  | Alternative to using `className`. Applied to the root element                            |
-| `& .RaList-actions`              | Applied to the actions container                                                         |
-| `& .RaList-main`                 | Applied to the main container                                                            |
-| `& .RaList-content`              | Applied to the child component inside the main container                                 |
-| `& .RaList-noResults`            | Applied to the component shown when there is no result                                   |
+The `<List>` component accepts the usual `className` prop but you can override many class names injected to the inner components by React-admin thanks to the `sx` property (as most Material UI components, see their [documentation about it](https://mui.com/customization/how-to-customize/#overriding-nested-component-styles)). This property accepts the following subclasses:
+
+| Rule name             | Description                                                                              |
+| --------------------- | ---------------------------------------------------------------------------------------- |
+| `& .RaList-root`      | Alternative to using `className`. Applied to the root element                            |
```

Comment:
I think that change is wrong: the `root` classname is applied to the root element, so there shouln't be a space. Try it in the List.stories.tsx to make sure.

Reply:
Your right.